src/client/components/app/Transactions/AddCreditPositionTx.tsx

```diff
@@ -52,7 +55,8 @@ export const AddCreditPositionTx: FC<AddCreditPositionProps> = (props) => {
   const { t } = useAppTranslation('common');
   const dispatch = useAppDispatch();
   const { allowVaultSelect, acceptingOffer, header, onClose, onPositionChange } = props;
-
+  const [transactionCompleted, setTransactionCompleted] = useState(false);
```

Comment:
I think this will be managed through the redux store. Will merge in new since u have other stuff in here but lets make a not for later